guides/source/configuring.md

```diff
@@ -172,7 +172,7 @@ Rails allows you to alter what generators are used with the `config.generators`
 ```ruby
 config.generators do |g|
   g.orm :active_record
-  g.test_framework :test_unit
+  g.test_framework :minitest
```

Comment:
This one yes. We can in the future change from `minitest` to `awesome_test` but we should not require users to change their config. In the end `test_unit` here means, the test framework included with Rails.